keps/sig-auth/3299-kms-v2-improvements/README.md

```diff
@@ -422,7 +429,8 @@ You can take a look at one potential example of such test in:
 https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
 -->
 
-N/A. When the feature is disabled, data stored in etcd will no longer be encrypted using the external kms provider with v2 API
+- When the feature is disabled, data stored in etcd will no longer be encrypted using the external kms provider with v2 API.
+- If the feature is disabled incorrectly (i.e without performing a storage migration), existing data that is encrypted with the external kms provider will be unable to be decrypted. This will cause list and get operations to fail for the resources that were encrypted.
```

Comment:
Should we explicitly call out "storage migration" in the "how to disable" section above?

Reply:
https://kubernetes.io/docs/tasks/administer-cluster/kms-provider/#disabling-encryption-at-rest has steps force all secrets to be decrypted

```bash
kubectl get secrets --all-namespaces -o json | kubectl replace -f -
```